src/v/raft/tests/BUILD

```diff
@@ -198,3 +198,427 @@ redpanda_cc_gtest(
         "//src/v/test_utils:gtest",
     ],
 )
+
+redpanda_cc_btest(
```

Comment:
Can we just use a single CPU here? I know we allow 8 in the cmake version, but this doesn't use any extra shards besides 0